sdk/communication/AzureCommunicationChat/Source/Custom/AzureCommuncationChatClient+Convenience.swift

```diff
@@ -38,7 +38,7 @@ extension AzureCommunicationChatClient {
     public func listChatReadReceipts(
```

Comment:
I would just add that you could just move this file into `Source` since anything not in `Source/Generated` is considered to be custom.